Automattic/sensei
# Change sensei admin menu to have Home as the default menu

Fixes #5904

### Changes proposed in this Pull Request

* It updates the admin menu structure in order to make the Home the main item of the Sensei menu.
* It also tweaks URLs, and screen IDs comparison that was changed because of the new menu structure.
* I fixed the things mostly navigating through all Sensei admin pages, testing the features, and searching extra cases with the following searches in the codebase:
  * `"course_page_"`
  * `"?post_type=course"`

### Testing instructions


* Unfortunately, it's a big change, and there's no specific tests instructions. To test it, we need to navigate through all menus under Sensei LMS in WP Admin, and make sure it continues working properly.

### Context

p6rkRX-4Mv-p2
